website/src/components/index/hero.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+import { OSSBadge } from '@site/src/components/global/oss-badge';
+import NpmCopy from '@site/src/components/global/npm-copy';
+import Wrapper from '@site/src/components/global/wrapper';
+import styles from './hero.module.scss';
+
+type NavItem = {
+  text: string;
+  url: string;
+};
+
+function NavItem({ text, url }: NavItem) {
+  const isExternalUrl = /^(http|https):\/\//.test(url);
+  const isDocsUrl = /^docs\//.test(url);
```

Comment:
Might consider using [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) to do these checks. (May or may not be worth the change.)